server/src/main/java/org/apache/druid/server/coordination/ChangeRequestsSnapshot.java

```diff
@@ -43,6 +43,11 @@ public static <T> ChangeRequestsSnapshot<T> fail(String resetCause)
     return new ChangeRequestsSnapshot<>(true, resetCause, null, null);
   }
 
+  public static <T> ChangeRequestsSnapshot<T> shutdown(String resetCause)
```

Comment:
this seems unused - did you decide to send ISE directly instead of this due to some reason?

Reply:
oops, yes this isn't used, was experimenting with different ways to handle shutdown, either failing the future or sending a "successful" shutdown, this is the latter, I'll remove it